UpdateWebviewServicesMetadata/handler.ts

```diff
@@ -0,0 +1,211 @@
+import { Context } from "@azure/functions";
+import { ServiceScopeEnum } from "@pagopa/io-functions-commons/dist/generated/definitions/ServiceScope";
+import {
+  RetrievedService,
+  ServiceModel,
+  ValidService
+} from "@pagopa/io-functions-commons/dist/src/models/service";
+import { errorsToReadableMessages } from "@pagopa/ts-commons/lib/reporters";
+import {
+  NonEmptyString,
+  OrganizationFiscalCode
+} from "@pagopa/ts-commons/lib/strings";
+import { enumType } from "@pagopa/ts-commons/lib/types";
+import { pipe } from "fp-ts/lib/function";
+import * as t from "io-ts";
+import * as E from "fp-ts/lib/Either";
+import * as TE from "fp-ts/lib/TaskEither";
+import * as O from "fp-ts/lib/Option";
+import * as T from "fp-ts/lib/Task";
+
+const ServiceExportCompact = t.interface({
+  i: NonEmptyString,
+  n: NonEmptyString,
+  q: t.number
```

Comment:
Can't we rename these with more meaningful ones?

Reply:
Is the purpose of this JSON prop to be minimized to reduce the size of the output JSON consumed by the website.
I can add some comments into the code, but the prop name must be 1 character long.

Reply:
Fixed in 601fbdf